docs/en/quick-answers/troubleshooting/certificate-fail-local-error-message.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Getting CERTIFICATE_VERIFY_FAILED when using {% include product %} Desktop on a local {% include product %} site
+pagename: certificate-fail-local-error-message
+lang: en
+---
+
+# Getting CERTIFICATE_VERIFY_FAILED when using {% include product %} Desktop on a local {% include product %} site
+
+## Use cases:
+When using a local install of {% include product %}, this error can arise in two scenarios:
+
+- when logging in {% include product %} Desktop
+- when downloading media from the Toolkit AppStore
+
+## How to fix:
+To solve this issue, need to provide a file to the {% include product %} API that contains the list of all valid CAs, including your own. We usually recommend that people download a fresh copy of [this file](https://github.com/certifi/python-certifi/blob/master/certifi/cacert.pem) from Python's `certifi` package as a starting point and then add their own CA at the end of the file. Then, save that file in a location all your users can access. Finally, on each computer, set the `SHOTGUN_API_CACERTS` environment variable to the full path to that file, for example `/path/to/my/ca/file.pem`.
```

Comment:
To solve this issue, *you* need to provide... (missing "you")

Reply:
Thank you! Fixed!